### ISSUE TITLE
Add a confirm pop-up after 'delete'

### ISSUE THREAD

**Issue by JPMSoftwire** [garethed/Q3#59]
So it turns out that Q3-geddon on Monday was caused by Kandyce repeatedly deleting Q's, as she thought that clicking 'Delete' was local to her machine.

I'd suggest adding a '"Delete this queue for everyone?" pop-up after the 'Delete' option so that we don't have this happen again.

**Comment by garethed** [comment]
Yes, I thought about this, but it's just annoying for everyone else who uses it correctly. I don't think I'll make any changes unless anyone else has a similar problem. Also I do have better logging now so I can see who's causing problems next time..

**Comment by JPMSoftwire** [comment]
Perhaps give an option to turn off the pop-ups after the first time? But if there's better logging, then that solves the problem, so I don't mind much - happy for you to close this either way.

**Comment by JPMSoftwire** [comment]
Or maybe even just adjusting the text to 'Delete Queue for Everyone'
